app/components/login_button_component.html.erb

```diff
@@ -3,5 +3,12 @@
       **tag_options,
       class: css_class,
     ) do %> 
-  Sign in with <%= content_tag(:span, APP_NAME, class: 'login-button__logo') %>
-<% end %>
\ No newline at end of file
+  Sign in with
+  <div class="login-button__logo">
+    <% if color != "primary-darker" %>
+      <svg aria-label="Login.gov" width="<%= width %>" height="<%= height %>" viewBox="0 0 293 41"><path fill="#fff" d="M7.44 6.89h17.2v22.06H7.44z"/><path fill="#112e51" d="M53.45 5.28h-6.16v27.31h17.87v-5.38H53.45zm30.76-.19c-8.35 0-14.67 5.89-14.67 13.85s6.32 13.93 14.67 13.93 14.67-6 14.67-13.93S92.56 5.09 84.21 5.09m.08 22.39a8.49 8.49 0 0 1-8.43-8.54 8.4 8.4 0 0 1 8.43-8.47 8.29 8.29 0 0 1 8.27 8.47 8.36 8.36 0 0 1-8.27 8.54m35.36-17.05a11.43 11.43 0 0 1 7.49 3l3.47-4.41a17.2 17.2 0 0 0-11-4c-8.51 0-14.9 6-14.9 13.85s6.29 13.99 14.58 13.99a20.2 20.2 0 0 0 11.08-3.55V17.57h-10.11v4.82H125V26a12.4 12.4 0 0 1-5.34 1.48 8.52 8.52 0 1 1 0-17zm19.61-5.15h6.17v27.31h-6.17zm34.53 17.56L160.52 5.28h-5.61v27.31h5.89V15.07l13.22 17.52h5.62V5.28h-5.85zm17.47 2.85a3.39 3.39 0 0 0-3.47 3.56 3.47 3.47 0 1 0 6.94 0 3.36 3.36 0 0 0-3.47-3.56m23.15-15.26a11.43 11.43 0 0 1 7.49 3l3.47-4.41a17.2 17.2 0 0 0-11-4c-8.51 0-14.9 6-14.9 13.85s6.28 13.93 14.55 13.93a20.2 20.2 0 0 0 11.08-3.55V17.57H215v4.82h4.77V26a12.4 12.4 0 0 1-5.34 1.48 8.52 8.52 0 1 1 0-17zm32.23-5.34C238.29 5.09 232 11 232 18.94s6.32 13.93 14.67 13.93 14.67-6 14.67-13.93S255 5.09 246.64 5.09m.08 22.39a8.49 8.49 0 0 1-8.43-8.54 8.4 8.4 0 0 1 8.43-8.47 8.29 8.29 0 0 1 8.28 8.47 8.36 8.36 0 0 1-8.28 8.54m39.02-22.2-7.49 20.6-7.45-20.6h-6.63l10.73 27.31h6.36L292.1 5.28z"/><path fill="#e21d3e" d="m11.38 25.34 1.45-8.28a4.39 4.39 0 0 1 2.71-7.86V0H4.34A4.34 4.34 0 0 0 0 4.34v21.28a2.6 2.6 0 0 0 .26 1.16c.94 1.9 4.65 8 15.27 12.69V25.7a24.3 24.3 0 0 1-4.15-.36"/><path fill="#b51e23" d="M26.72 0H15.53v9.19A4.39 4.39 0 0 1 18.28 17l1.45 8.32a24.3 24.3 0 0 1-4.2.36v13.79C26.14 34.8 29.86 28.68 30.8 26.79a2.6 2.6 0 0 0 .26-1.16V4.34A4.34 4.34 0 0 0 26.72 0"/></svg>
```

Comment:
Where do these images come from? Are they the same as `login-gov-logo.svg` and `login-gov-logo-reverse.svg`? I'm wondering if we can read this SVG data from the design system assets if so

Reply:
We need them to be in the html file because when we link to them in the stylesheet and then embed the stylesheet within the HTML its grabbing the wrong path for the image assets (since the build changes the name of the file).

Reply:
Yes, we do the same thing in the security key image component: https://github.com/18F/identity-idp/blob/904a151e16b88059a04025df68c34c5761a46a3c/app/components/security_key_image_component.rb#L52-L57

We can even modify the SVG in-memory  (ex setting attributes like `aria-label` if we want) instead of having to template the changes

Reply:
Ah, that makes sense! I will try that.

Reply:
I think I got that working - thank you @zachmargolis  its much more elegant now